packages/@o3r/styling/schematics/theming-to-component/index.ts

```diff
@@ -0,0 +1,62 @@
+import {
```

Comment:
(Not the good place for the comment but...)
Actually the `ng-add` of the `@o3r/styling` (and of `@o3r/configuration`) packages should edit the `generators` configuration of the `angular.json` to set the component generator option `useOtterTheming` to `true`